packages/blockchain/test/index.spec.ts

```diff
@@ -38,6 +38,64 @@ tape('blockchain test', (t) => {
     st.end()
   })
 
+  t.test('should initialize correctly with consensus option passed in', async (st) => {
+    const common = new Common({ chain: Chain.Goerli, hardfork: Hardfork.Chainstart })
+    let blockchain = await Blockchain.create({ common: common, consensus: 'casper' })
+    let iteratorHead = await blockchain.getIteratorHead()
+    st.ok(
```

Comment:
Can we revise these tests to confirm that the instantiated `blockchain` object contains the expected consensus algorithm?  Also, I see you're using `goerli` as the chain.  It would be good to validate that `common.consensusAlgorithm` matches the `consensus` class instantiated with the blockchain.  Since Goerli is PoA, we wouldn't want to add an ethash consensus class to the blockchain.